api/src/endpoints/svgThumbnail.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import fs from 'fs';
+import getSvgThumbnail from 'utils/2d-map';
+
+const routes = express.Router();
+const sharp = require('sharp');
+let HPA_JSON = null;
+
+routes.get('/:svgName', async (req, res) => {
+  const { svgName } = req.params;
+  const { model } = req.query;
```

Comment:
I think a way to check that the query model is valid is already implemented in the API, so it would be good to make use of that.

Reply:
Thanks for pointing out this. This comment has been addressed in the commit #db510182.
The version of the model is also parsed when handling the query parameters with the commit #a6046c95